src/select/select__popup.tsx

```diff
@@ -703,7 +707,8 @@ export default class SelectPopup<T = unknown> extends PureComponent<SelectPopupP
   tags: PropTypes.object,
   toolbar: PropTypes.node,
   topbar: PropTypes.node,
-  top: PropTypes.number
+  top: PropTypes.number,
+  overscrollListBehaviorNone: PropTypes.bool
```

Comment:
I suggest naming this prop in a less technical way, such as `preventListOverscroll`.